releasenotes/notes/fix-ddtrace-file-logs-bytes-env-var-ed690ffe3fb96f7e.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
is this needed? we haven't released 1.3 yet so messaging should be required

Reply:
1.3 has now been released